src/main/resources/application.yaml

```diff
@@ -57,8 +84,15 @@ application:
     enable: false
     url:
     port:
-  cors:
-    allowed-origins:
+  public-urls:
+    - /swagger-ui/**
+    - /v3/api-docs/**
+    - /api/healthcheck
+    - /
+  roles:
+    admin: 'admin'
+    designer: 'designer'
+  corsOrigins: '*'
```

Comment:
corsOrigins should be empty here. There's a check on this property when loading app to force setting this property